src/envoy/http/jwt_auth/sample/APToken/aptoken-envoy.conf

```diff
@@ -0,0 +1,116 @@
+{
```

Comment:
Does this file used in unit test? I don't see any code using this.

Reply:
This file is used in a guide I added.  I have added a guide of sending an example exchanged token to the jwt-authn filter and the Istio authn filter, and observing that the example backend echoes back the request when the authentication succeeds.

Reply:
It's better to write a test. You can add them to test/integration. There is already few test setup examples.

Reply:
A test has been added.